### ISSUE TITLE
Suggestion for custom-named resource replacement error

### ISSUE THREAD

**Issue by pinnareet** [seed-run/homepage#17]
I have been working with AWS SAM and ran into a similar error here: [CloudFormation cannot update a stack when a custom-named resource requires replacing](https://github.com/seed-run/homepage/blob/master/_docs/serverless-errors/cloudformation-cannot-update-a-stack-when-a-custom-named-resource-requires-replacing.md). AWS support recommended that I remove the resource from the template so it gets deleted, then put it back in later. I think it's cleaner that way than renaming the resource to a dummy new name. Well, just an idea :) I'm not entirely sure if things will work the same way in Serverless. Would like to hear your thoughts on this. Thanks.

**Comment by jayair** [comment]
Yeah that could work as well. The issue is if your stack has a lot of other resources that you can't afford to remove right away. The rename trick just targets the one resource that has the problem.
